cmd/collector/app/builder/builder_flags.go

```diff
@@ -38,4 +38,6 @@ var (
 	CollectorPort = flag.Int("collector.port", 14267, "The tchannel port for the collector service")
 	// CollectorHTTPPort is the port that the collector service listens in on for http requests
 	CollectorHTTPPort = flag.Int("collector.http-port", 14268, "The http port for the collector service")
+	// CollectorHealthCheckHTTPPort is the port that the health check service listens in on for http requests
+	CollectorHealthCheckHTTPPort = flag.Int("collector.health-check-http-port", 15268, "The http port for the health check service")
```

Comment:
No reason, I just added 1000 to the other HTTP port. I could add +1 instead of +1000, though.